### DISCUSSION TITLE
Is there any way to enable Advanced Settings on the M1 air internal display?

### DISCUSSION THREAD

**Discussion by quarkw** [MonitorControl/MonitorControl#1322] [Q&A]
I want to do combined hardware/software dimming and control the min/max and curves

**Comment by waydabber** [comment]
You can use combined dimming with [BetterDisplay](https://betterdisplay.pro) when using an Apple display. With MC, combined dimming works with external, DDC controlled displays only.

**Reply by quarkw** [reply]
Thanks for the suggestion! I did take a look at BetterDisplay but I don't think it gives the granular control that I want for hardware brightness limits and curves.

**Reply by waydabber** [reply]
Yes, for the Apple hardware brightness control it does not add the rescaling options, only for DDC. That's why I was referring to combined dimming only. Apple's displays are usually calibrated well and linear brightness control just works great using native APIs. Btw, why do you want to change the curves and limits on it?

**Reply by quarkw** [reply]
It's because of [PWM fatigue](https://www.notebookcheck.net/Why-Pulse-Width-Modulation-PWM-is-such-a-headache.270240.0.html) which can cause eye strain and headaches.

I think all of the LED macbooks (which includes the m1 air and mb pro) use PWM to control brightness. They have a fairly high refresh rate so I think I would be ok going to about 50% of the hardware brightness.
